UnusualCombat.tex

```diff
@@ -11,35 +11,37 @@ \part*{\begin{center}
 }
 \section{Item interactions \small (PHB p.190)}
 You can interact with one item or feature of the environment for free, during your movement or action.
-A second interaction requires an action.\\
+A second interaction requires an action.
+\vspace{3mm}
+
 Examples:
 \begin{itemize}
-\item draw / sheathe weapon
-\item open door / pull lever / extinguish small flame
-\item take an item from a container / pick up an item
-\item stuff something into your mouth / drink something
-\item hand someone something / accept something from someone
-\end{itemize} 
+\item Draw or sheathe your weapon
+\item Open a door, pull a lever or extinguish a small flame
+\item Take an item from a container or pick it up
+\item Stuff something into your mouth or drink something
+\item Hand someone something or accept something from someone
+\end{itemize}
 \section{Dropping prone \small (PHB p.190 + 292)}
 \begin{itemize}
-\item You can drop prone for free
+\item You can drop prone without any costs
 \item Standing back up requires half your movement
 \item You cannot stand up when your movement is 0
-\item While prone, you can only crawl, costing 2ft for every ft of movement and you have disadvantage on attack rolls
-\item An attack against you has advantage when the attacker is within 5ft otherwise the attack has disadvantage
+\item While prone, you can only crawl, costing 2 ft for every foot of movement and you have disadvantage on attack rolls
+\item An attack against you has advantage when the attacker is within 5 ft otherwise the attack has disadvantage
 \end{itemize}
 \section{Search \small (PHB p.193)}
-You stop paying attention to the combat and instead focus on finding something, depending on the nature of your search, you DM may have you roll a Perception(WIS) or Investigation(INT) check.
+You stop paying attention to the combat and instead focus on finding something, depending on the nature of your search, your DM may have you roll a Perception (WIS) or Investigation (INT) check.
 \section{Special melee attacks \small (PHB p.195)}
 Grapple and shove:
 \begin{itemize}
 \item Replace one attack against a target within reach
 \item The target must be no more than 1 size larger than you
-\item Make an Athletics(STR) check against the opponents Athletics(STR) or Acrobatics(DEX) check instead of an attack roll
+\item Make an Athletics (STR) check against the opponents Athletics (STR) or Acrobatics (DEX) check instead of an attack roll
 \end{itemize}
 Grapple:
 \begin{itemize}
-\item You must have one free hand
+\item You must have one hand free
```

Comment:
"One free hand" is correct english grammar and is more natural than "having one hand free"

Reply:
Both sentences do not give any errors when parsing them through a grammar checker, so I guess it's more a thing of personal preference.